src/Provider/IndexUrlProvider.php

```diff
@@ -33,7 +33,7 @@ public function addProvider(UrlProviderInterface $provider): void
         $this->providers[] = $provider;
     }
 
-    public function generate(): iterable
+    public function generate(): array
```

Comment:
Why has this changed? An `iterable` is quite different from an `array`, and can be useful sometimes. It doesn't have to be an `array` imo.